### PR TITLE
fix: react-editor 'onEvent' props missing

### PR DESCRIPTION
**Pull request by yar2001**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] It does not introduce a breaking change or has a description of the breaking change

### Description
Fix React-editor 'onEvent' props missing, or it will cause a fatal error when using TypeScript.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/react-editor/index.d.ts [hunk 1]
```diff
@@ -2,7 +2,13 @@ import { Component } from 'react';
 import ToastuiEditor, { EditorOptions } from '@toast-ui/editor';
 import ToastuiEditorViewer, { ViewerOptions } from '@toast-ui/editor/dist/toastui-editor-viewer';
 
-type EditorProps = Omit<EditorOptions, 'el'>;
+type EditorProps = Omit<EditorOptions& {
+  onLoad?: (param: Editor) => void;
+  onChange?: (param: { source: SourceType | 'viewer'; data: MouseEvent }) => void;
+  onStateChange?: (param: MarkdownToolbarState | WysiwygToolbarState) => void;
+  onFocus?: (param: { source: SourceType }) => void;
+  onBlur?: (param: { source: SourceType }) => void;
```
Comment (seonim-ryu) [inline apps/react-editor/index.d.ts:10]: `SourceType`, `MarkdownToolbarState`, `WysiwygToolbarState` Where did you get these types from?

**Review by seonim-ryu — CHANGES_REQUESTED** [review]
@yar2001 I've completed the review and would appreciate it if you check the comment.

File: apps/react-editor/index.d.ts (hunk 1 above)
Comment (yar2001) [reply apps/react-editor/index.d.ts:10]: > `SourceType`, `MarkdownToolbarState`, `WysiwygToolbarState` Where did you get these types from?

It is from apps/editor/index.d.ts 191
```
  interface EventMap {
    load?: (param: Editor) => void;
    change?: (param: { source: SourceType | 'viewer'; data: MouseEvent }) => void;
    stateChange?: (param: MarkdownToolbarState | WysiwygToolbarState) => void;
    focus?: (param: { source: SourceType }) => void;
    blur?: (param: { source: SourceType }) => void;
  }
```

The doc said that in React component, the user could just add *on* before the event name to pass callback funtion. And I 
 speculate that the types is similiar to the types in the main editor.

Comment (yar2001) [reply apps/react-editor/index.d.ts:10]: I wish there were some more elegant ways to do it, but it seems I have no choice but to do so.

Comment (seonim-ryu) [reply apps/react-editor/index.d.ts:10]: @yar2001 Sorry for the late answer. You can `import` the editor type, but when I check the [Editor's type declaration file](https://github.com/nhn/tui.editor/blob/master/apps/editor/index.d.ts),  it is not exported. If possible, how about modifying the editor's declaration file as well?

For example, you can modify the `SourceType` type as follows. You can process other types like this.

> e.g. https://github.com/nhn/tui.editor/blob/301f5e6ac36b75dfc6bb40eb35fcb3b474fb8142/apps/editor/index.d.ts#L189

```js
- type SourceType = 'wysiwyg' | 'markdown';
+ export type SourceType = 'wysiwyg' | 'markdown';
```

**Comment by yar2001** [comment]
Done! This works great in my project.

**Review by seonim-ryu — APPROVED** [review]
@yar2001 
I checked the content that reflected the review too late. Sorry.. I checked the work and I'll merge it. Thanks for your contributing! 👍

**Pull request merged**